lte/gateway/python/integ_tests/s1aptests/test_attach_ipv4v6_pdn_type.py

```diff
@@ -29,12 +29,16 @@ def test_attach_ipv4v6_pdn_type(self):
             capable """
         # Set PDN TYPE to IPv4V6 i.e. 3. IPV4 is equal to 1
         # IPV4orIPv6 is equal to 3 in value
```

Comment:
The comment makes no sense as it says both IPv4v6 and IPv4orIPv6 has enum value 3.